org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/commands/ProjectCommand.java

```diff
@@ -128,6 +130,17 @@ public static boolean isTestFile(String uri) throws CoreException {
 		return false;
 	}
 
+	public static List<URI> getAllJavaProjects() {
+		List<URI> javaProjects = new LinkedList<>();
+		for (IJavaProject javaProejct : ProjectUtils.getJavaProjects()) {
```

Comment:
javaProject

Reply:
Hmm, I think I need to install a spelling checking extension. 😄